resource_activity/models/resource_available.py

```diff
@@ -12,9 +12,14 @@ class ResourceAvailable(models.Model):
     _description = "Resource Available"
 
     name = fields.Char(related="resource_id.serial_number", string="Name")
-    resource_id = fields.Many2one("resource.resource", string="Resource", required=True)
+    resource_id = fields.Many2one(
+        comodel_name="resource.resource", string="Resource", required=True
+    )
     registration_id = fields.Many2one(
-        "resource.activity.registration", string="Registration"
+        comodel_name="resource.activity.registration", string="Registration"
+    )
+    allocation_id = fields.Many2one(
+        comodel_name="resource.allocation", string="Allocation"
     )
```

Comment:
to manage existing records, a migration script should set this field from the `resource_id` and `registration_id`.